package.json

```
@@ -122,7 +122,7 @@
     "dev:with-companion": "npm-run-all --parallel start:companion dev",
     "dev": "yarn workspace @uppy-dev/dev dev",
     "lint:fix": "yarn run lint -- --fix",
-    "lint:markdown": "remark -f -q . -i .remarkignore",
+    "lint:markdown": "remark -f -q -i .remarkignore . .github",
```

Do you know why? Is it because `remark` always ignore dot-leading dirs? Anyway, I think this should be aligned with GHA list:
```suggestion
    "lint:markdown": "remark -f -q -i .remarkignore . .github/CONTRIBUTING.md",
```

---

i tried to find if there was any ignore of .github but no avail, so i can only assume that it ignores all dotfolders yes